cerbos/model.go

```diff
@@ -1212,29 +1212,39 @@ type PlanResourcesResponse struct {
 }
 
 type (
-	ListPoliciesOption func(*requestv1.ListPoliciesRequest)
+	FilterOptions struct {
+		NameRegexp      string
+		ScopeRegexp     string
+		VersionRegexp   string
+		IncludeDisabled bool
+	}
+	// FilterOption allows filtering policies while calling InspectPolicies and ListPolicies.
+	FilterOption func(*FilterOptions)
+	// ListPoliciesOption allows filtering policies while calling ListPolicies
+	// Deprecated: ListPoliciesOption is deprecated, use FilterOption instead.
+	ListPoliciesOption FilterOption
```

Comment:
I think this has to be `ListPoliciesOption = FilterOption` (a type alias). Please test using a program that uses `ListPoliciesOption` as a variable (`opts := []cerbos.ListPoliciesOption{...}`) and switching the dependency to the new version to make sure that the old code still compiles.

Reply:
Fixed the issue and tested it with the `cerbos/cerbos`. ([ref #1](https://github.com/cerbos/cerbos/blob/f08a3d8c6270994a9703fadbbe0dc51e77006b7b/cmd/cerbosctl/get/internal/policy/policy.go#L35), [ref #2](https://github.com/cerbos/cerbos/blob/f08a3d8c6270994a9703fadbbe0dc51e77006b7b/cmd/cerbosctl/get/internal/policy/policy.go#L38))